tests/ios/unit-test/BUILD.bazel

```diff
@@ -29,7 +29,8 @@ ios_unit_test(
         "empty.swift",
     ],
     minimum_os_version = "12.0",
-    test_host = "//rules/test_host_app:iOS-9.3-AppHost",
```

Comment:
Do we have another check with this versioned app host - otherwise it looks to drop coverage and would suggest to spin up another `ios_unit_test`

Reply:
Will add another test case. 👍